src/components/Button/Button.st.css

```diff
@@ -3,6 +3,28 @@
   -st-default: Button;
 }
 
+:import {
+  -st-from: "../../palette.st.css";
+  -st-named: main;
+}
 .root {
   -st-extends: Button;
-}
\ No newline at end of file
+  -st-states: skin(enum(standard)), priority(enum(primary)), size(enum(medium));
+
+  box-sizing: border-box;
+  
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.root:skin(standard):priority(primary) {
```

Comment:
what about hover, error, success handling?
next pr?

Reply:
next pr will be only for the styles which are needed for FLoatingHelper